src/hooks/useInput.ts

```diff
@@ -0,0 +1,19 @@
+import type { ChangeEvent } from 'react';
+import { useState } from 'react';
```

Comment:
@dongkyun-dev 
이후에 같은 경로에서 import될 때 type을 inline으로 작성할 지 아니면 위처럼 분리해 작성할지 
한 가지 방식으로 강제하는 rule을 추가하면 좋을 것 같습니다!

현재는 두가지 방식 다 작성 가능하도록 되어있더라고요.

```suggestion
import { useState, type ChangeEvent } from 'react';
```

Reply:
@wonjin-dev @dmswl98 
좋아요 혹시 선호하시는 방식 있으신가요?

Reply:
@dongkyun-dev 저는 현재 코드에서 많이 사용된 방식으로 하면 좋을 것 같아요.

Reply:
@dmswl98 
저는 개인적으로 분리하지 않고 하나의 라인에서 작성하는게 좋긴 한데....! 관련해서 린트룰을 찾아볼까요?

Reply:
@dongkyun-dev 그럼 이렇게 작성하면 됩니다!!
```cjs
'@typescript-eslint/consistent-type-imports': [
   'error',
   { fixStyle: 'inline-type-imports' },
],
```

Reply:
@dmswl98 
오오 혹시 반영해서 PR 작성해주실 수 있나요?